colortv/docs/androidTVSDK.md

```diff
@@ -591,83 +591,214 @@ public class MainActivity extends Activity {
-setSnapEnabled(boolean enabled)
-resetToDefault()
+ColorTvRecommendationsController recommendationsController = ColorTvSdk.getRecommendationsController();
+ColorTvContentRecommendationConfig recommendationConfig = recommendationsController.getConfig();
 ```
 
-All methods (except for `setSnapEnabled` - only mobile) are applicable for TV, mobile and tablet devices.
+Using the config object, you can access the following methods:
+
+```java
+recommendationConfig.setGridLayout(Device device, @LayoutRes int layoutResId);
+recommendationConfig.setItemLayout(Device device, @LayoutRes int layoutResId);
+recommendationConfig.setRowCount(Device device, int rowCount);
+recommendationConfig.setFont(Device device, Typeface typeface);
+recommendationConfig.setSnapEnabled(boolean enabled);
+recommendationConfig.resetToDefault();
+```
 
+All methods (except for `setSnapEnabled` - only mobile) are applicable to TV, mobile and tablet devices. `Device` enum is nested inside the `ColorTvRecommendationConfig` class.
 
 #### setGridLayout(Device device, @LayoutRes int layoutResId)
 
 This method is used to set custom grid layout for specified device type. You can add additional images, texts etc. We will handle the following views if they're available:
 
-| ID                              | View type  | Animation    | Description                                     | Device |
-|---------------------------------|------------|--------------|-------------------------------------------------|--------|
-| ctv_rvGrid                      | RecyclerView |              |                                                 | ALL    |
-| ctv_ivFavoriteContainer         | Any layout | Visibility.VISIBLE and scale | Appears on play/pause button click if available | ALL    |
-| ctv_featuredUnitLayoutContainer | Any layout |              | Used to inject featured content if available    | TV     |
-| ctv_ivGridClose                 | ImageView  | Scale        |                                                 | TV     |
-| ctv_layoutSubscriptionFragment  | Any layout |              | Discovery center only                           | TV     |
+<details><summary>Available IDs</summary>
+
+
+#### **ID:** ctv_rvGrid
+* **View type:** `android.support.v7.widget.RecyclerView`
+* **Description:** Displaying individual recommendations.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_ivFavoriteContainer
+* **View type:** Any view extending the View class (eg. `LinearLayout`, `ImageView`).
+* **Description:** Appears when user presses the play-pause remote button on TV or taps on `ImageView` with id `ctv_ivFavoriteIcon` on Mobile/Tablet if available.
+* **Animation:** Is displayed and scales up for 1 second.
+* **Device:** ALL
+![Favorite button click](images/favorite_button_click_android.gif)
+
+
+#### **ID:** ctv_featuredUnitLayoutContainer
+* **View type:** Any view extending the View class (eg. `LinearLayout`, `ImageView`), `FrameLayout` is recommended.
```

Comment:
Still imo it would be good to describe here what it is (as far as I know it won't be just ads).